core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
Ah yes, the file was moved. I have fixed the link, thanks for reporting. (That said, the thing to do in that case, in addition to reporting it, is to look around for other directive files to get a sense of what the existing styling requirements are.)

Also, please remember to address the point in my top-level comment below (about the tradeoff between colors and speed).